packages/interactivity/src/vdom.ts

```diff
@@ -127,18 +131,22 @@ export function toVdom( root ) {
 							SCRIPT_DEBUG === true
 						) {
 							// eslint-disable-next-line no-console
-							console.warn( `Invalid directive: ${ name }.` );
+							console.warn(
```

Comment:
Let's wait for https://github.com/WordPress/gutenberg/pull/61409 to be merged and update with its function.